src/WebJobs.Script.WebHost/Security/DefaultKeyValueConverterFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public sealed class DefaultKeyValueConverterFactory : IKeyValueConverterFactory
+    {
```

Comment:
I originally had the encryption changes as part of what I was preparing for this PR. In addition to aiding with tests, this factory will be responsible for providing the appropriate converter when the encryption converter is re-added.